README.md

```diff
@@ -37,45 +37,190 @@ However, processing WSIs is far from being trivial. First of all, WSIs can be st
+* ```breast_tissue_diagnostic_green_pen```: Breast invasive carcinoma (TCGA-BRCA) diagnostic slide with green pen marks
+* ```breast_tissue_diagnostic_red_pen```: Breast invasive carcinoma (TCGA-BRCA) diagnostic slide with red pen marks
+* ```breast_tissue_diagnostic_black_pen```: Breast invasive carcinoma (TCGA-BRCA) diagnostic slide with black pen marks
+
+The slides will automatically be downloaded and saved in a temporary cached directory in your local machine.
+
+**NB** To use the data module, you need to install ```pooch```.
+
+Each data function outputs the corresponding slide as an OpenSlide object, and the path where the slide has been saved:
+
+
+```python
+breast_svs, breast_path = breast_tissue()
+heart_svs, heart_path = heart_tissue()
+```
+
+### Slide
+
+
+```python
+from histolab.slide import Slide
+```
+
+Convert the slide into a ```Slide``` object. ```Slide``` takes as input the path where the slide is stored and the ```processed_path``` where the thumbnail and the tiles will be saved.
+
+
+```python
+breast_slide = Slide(breast_path, processed_path='processed')
+heart_slide = Slide(heart_path, processed_path='processed')
+```
+
+As a ```Slide``` object, you can now easily retrieve information about the slide, such as the slide name, the dimensions at native magnification, the dimensions at a specified level, save and show the slide thumbnail, or get a scaled version of the slide.
+
+
+```python
+print(f"Slide name: {breast_slide.name}")
+print(f"Dimensions at level 0: {breast_slide.dimensions}")
+print(f"Dimensions at level 1: {breast_slide.level_dimensions(level=1)}")
+print(f"Dimensions at level 2: {breast_slide.level_dimensions(level=2)}")
+```
+
+    Slide name: 9c960533-2e58-4e54-97b2-8454dfb4b8c8
+    Dimensions at level 0: (96972, 30681)
+    Dimensions at level 1: (24243, 7670)
+    Dimensions at level 2: (6060, 1917)
+
+
+
+```python
+print(f"Slide name: {heart_slide.name}")
+print(f"Dimensions at level 0: {heart_slide.dimensions}")
+print(f"Dimensions at level 1: {heart_slide.level_dimensions(level=1)}")
+print(f"Dimensions at level 2: {heart_slide.level_dimensions(level=2)}")
+```
+
+    Slide name: JP2K-33003-2
+    Dimensions at level 0: (32671, 47076)
+    Dimensions at level 1: (8167, 11769)
+    Dimensions at level 2: (2041, 2942)
+
+
+
+```python
+breast_slide.save_thumbnail()
+print(f"Thumbnails saved at: {breast_slide.thumbnail_path}") 
+heart_slide.save_thumbnail()
+
+print(f"Thumbnails saved at: {heart_slide.thumbnail_path}") 
+```
+
+    Thumbnails saved at: processed/thumbnails/9c960533-2e58-4e54-97b2-8454dfb4b8c8.png
+    Thumbnails saved at: processed/thumbnails/JP2K-33003-2.png
+
+
+
+```python
+breast_slide.show() 
+heart_slide.show()
+```
+
+![caption](images/thumbnails.png)
+
+### Tiles extraction
+
+Now that your slide object is defined, you can automatically extract the tiles. ```RandomTiler``` method crops random tiles from the slide.
```

Comment:
`RandomTiler` is not a method. You could say: A `RandomTiler` object crops...

(note the A before)